components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -555,6 +574,15 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
     this.store.dispatch(new UpdateNodeFilters({filters: nodeFilters}));
   }
 
+  isCurrentPageEmpty(nodes: Node[], loadedStatus: EntityStatus): boolean {
+    return loadedStatus === EntityStatus.loadingSuccess && nodes.length === 0;
+  }
+
+  // If there is a page parameter in the URL the first page is not selected.
+  isNotFirstPage(allUrlParameters: Chicklet[]): boolean {
+    return allUrlParameters.findIndex((parm: Chicklet) => parm.type === 'page') >= 0;
```

Comment:
[nit] How about [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)? It's pretty much the same except that it's already a boolean:
```suggestion
    return allUrlParameters.some((parm: Chicklet) => parm.type === 'page');
```

Reply:
Thank you! 
I always forget about 'some'. I wish there was an 'exists'.